mattermost-plugin/webapp/src/components/rhsChannelBoards.test.tsx

```diff
@@ -16,6 +16,11 @@ import RHSChannelBoards from './rhsChannelBoards'
 jest.mock('../../../../webapp/src/octoClient')
 const mockedOctoClient = mocked(octoClient, true)
 
+jest.mock('react', () => ({
```

Comment:
I'm not sure about this kind of solution to show the content. My perception is you are mocking here the wrong thing, my expectation is that the mocking of the client should be enough and just using `act` around the rendering should wait until the effect is executed.

Reply:
by "mocking of the client" do you mean mocking the dispatched requests? and then with act make sure the state change has been already executed and so the subsequent re-render?